spec/api_spec.rb

```diff
@@ -21,10 +21,10 @@ class Bar; end
       end
 
       context "when setting custom options" do
-        before { subject.setup foo: { bar: "baz" }, url: "https://api.example.com" }
+        before { subject.setup foo: { bar: "baz" }, url: "https://api.example.com", timeout: 10, open_timeout: 2 }
```

Comment:
So the build is passing on Travis w/ your [first commit](https://travis-ci.org/remiprev/her/builds/240388060) and  to be sure I tested on a separate [branch](https://travis-ci.org/remiprev/her/builds/240842291). We should be ok  to drop commit 6f01adce19caa36c24b4a7ec931e99aa6fc4578d 👍